WORKSPACE.bzlmod

```diff
@@ -45,7 +45,7 @@ git_repository(
     remote = "https://github.com/facebookincubator/fbjni",
 )
 
-RN_VERSION = "0.74.1"
+RN_VERSION = "0.78.1"
```

Comment:
Could we provide a changelog for what's changed between these versions? At least, how it changes Hermes implementation/execution.